guide/payments/hide-sensitive-info.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: guide
+title: Hide Sensitive Information
+nav_order: 71
+has_children: false
+parent: Payments
+permalink: /guide/payments/hide-sensitive-information/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/hide-sensitive-information-header.png
+
+---
+{% include picture.html
+   image = "/assets/images/payments/hide-sensitive-information-header.png"
+   retina = "/assets/images/payments/hide-sensitive-information-header.png"
+   mobile ="/assets/images/payments/hide-sensitive-information-header.png"
+   mobileRetina ="/assets/images/payments/hide-sensitive-information-header.png"
+   alt-text = "Payment Privacy & Security"
+   caption = 'Person by [DrawKit](https://www.drawkit.io/illustrations/mobile-article-colour){:target="_blank" rel="nofollow"} and trees by [Flaticon](https://www.flaticon.com/free-icon/tree-with-many-leaves_25267){:target="_blank" rel="nofollow"}'
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Hiding Sensitive Information
+
+Imagine this scenario. You are in a public place, and you need to make a payment using your bitcoin wallet. You open your wallet on your phone, but you don’t feel comfortable having your address and balance information clearly visible to strangers who may be looking over your shoulder. Hence by giving users the ability to hide sensitive information in their wallet if desired, they gain an added sense of privacy and security when using the app in public.
```

Comment:
> clearly visible to strangers who may be looking over your shoulder

Video surveillance can be another attack vector to add

Reply:
That makes an excellent point! considering that video surveillance is quite common nowadays.